mergify_engine/queue/merge_train.py

```diff
@@ -539,6 +539,11 @@ async def _create_draft_pull_request(
                 ).json(),
             )
         except http.HTTPClientSideError as e:
+
+            if "errors" not in e.response.json():
```

Comment:
I find this cryptic. Can we add a comment at least?
Or even an example of the JSON you'd expect?

Reply:
I'd even check for the message in the JSON to be clear.

Reply:
Indeed referencing the expected json should do the trick